CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+X.Y.Z Release notes (YYYY-MM-DD)
+=============================================================
+### Enhancements
+* Realm JS now supports all Nodejs versions above Node 10.
+* Helper methods 'myinstance.keys()', 'myinstance.entries()' are now available to return the properties of that instance including the realm properties from the prototype.
+* An instance method 'toJSON()' is implemented on every Realm Object class to support 'JSON.stringify()' to work as expected on every instance
+
+### Breaking changes
+* Nodejs 8 is not supported anymore.
+* Realm objects properties are now defined as accessors on the instance prototype. (Previously they were defined as values on the instance)
+* Due to accessor properties change 'Object.keys()', 'Object.entries()', 'Object.getOwnPropertyDescriptors()' will not return the realm schema properties.
+
+
+### Fixed
+* ECMAScript 2015 Class syntax is fully supported now by moving all properties to be accessors on the instance prototype allowing Realm JS to invoke class constructors as constructors (using 'new' instead of previously calling them as functions with 'call')
+
+### Compatibility
+* None.
```

Comment:
?? Please copy section from earlier release.

Reply:
I don't see the issue. Can you elaborate ?

Reply:
@bmunkholm ^

Reply:
The content of the Compatibility section should not be "None." You must copy the content from the previous release (and adjust if needed).

Reply:
I see thanks